app/presenters/citable_attributes.rb

```diff
@@ -15,6 +15,10 @@
 #     # => A hash of attributes.
 #
 # Ported from Sufia: app/models/chf/citable_attributes.rb
+#
+# A citable_attributes instance can be passed to a CitationDisplay as follows:
+#
+# CitationDisplay.new(citable_attributes).display
 
```

Comment:
Do you think it would be a more sensible API to have `CitationDisplay.new(work).display`?  Let the `CitationDisplay` instantiate a CitableAttributes itself. I know this is different than how it worked in chf_sufia. What do you think?

Reply:
Ah, what I said may actually *be( the API of CitationDisplay below, this comment is just wrong?

Reply:
Oh yeah, we had the same idea and I refactored but forgot to change that comment. Will fix.

Reply:
Done.